core/src/main/java/org/aerogear/mobile/core/metrics/publisher/NetworkMetricsPublisher.java

```diff
@@ -32,14 +32,15 @@ public void publish(Metrics... metrics) {
             for (final Metrics m : metrics) {
                 json.put(m.identifier(), new JSONObject(m.data()));
             }
-
+            json.put("timestamp","TODO");
```

Comment:
Ideally we will need to refactor from Metrics[] to individual object that will contain metrics. and wrap that into json object. 
Presenting this as idea as we may not need that. 

We could replicate this back to proposal to make sure that we have this covered for other platforms.